lib/ember-orbit/links/has-one-object.js

```diff
@@ -1,17 +0,0 @@
-import LinkProxyMixin from './link-proxy-mixin';
-
-/**
- @module ember-orbit
- */
-
-/**
- A `HasOneObject` is an `ObjectProxy` that represents the contents of a has-one
- relationship.
-
- @class HasOneObject
- @namespace EO
- @extends Ember.ObjectProxy
- */
-var HasOneObject = Ember.ObjectProxy.extend(LinkProxyMixin);
```

Comment:
I was originally planning on using a LiveQuery to keep hasOnes updated but that would have required using a proxy for the hasOne. As @gnarf mentioned this is a bit of a pain as it causes `moon.get('planet') !== jupiter` and you have to resort to `moon.get('planet.id') !== jupiter.get('id')`.